projects/testing-library/package.json

```
@@ -32,13 +32,15 @@
     "@angular/common": ">= 14.0.0",
     "@angular/platform-browser": ">= 14.0.0",
     "@angular/router": ">= 14.0.0",
-    "@angular/core": ">= 14.0.0",
-    "rxjs": ">= 7.4.0"
+    "@angular/core": ">= 14.0.0"
```

We still want to register rxjs as a peer dep.
But, we also want to support v6

```suggestion
    "@angular/core": ">= 14.0.0",
    "rxjs": "^6.5.3 || ^7.4.0"
```

---

I think that would work for me.

I couldn't find any usage of rxjs in the library code itself. Would something not work as expected, if someone installed the library and doesn't provide rxjs? If not I would like to free the user of the obligation to install rxjs. What do you think?

---

Oh! That's a very good point 😅 I'm so used to having it as a peer dep, but you're right we can just remove it here.
We also don't need it in the dev deps, because that's being handled at the root level.

---

I am glad I didn't miss anything. Thanks for checking and responding so fast! 🙂 I updated the PR and removed it from dev deps as well.